### ISSUE TITLE
Newtonsoft.Json.JsonReaderException: Unexpected character encountered while parsing value

### ISSUE THREAD

**Issue by Ordnael92** [roro08/Sketchfab-Ripper#19]
![Immagine 2022-05-11 135322](https://user-images.githubusercontent.com/85753225/167843555-4f3c5183-a0ec-4636-b93c-147f0cb653e9.jpg)
This appears whenever I try to download anything.
It was working fine until yesterday!

**Comment by Korozion** [comment]
Same here. Looks like sketchfab has also modified their .binz files after encrypting their textures...
unless someone updates the ripper, I believe this is it for Sketchfab-Ripper :/

**Comment by rasamaya** [comment]
Me too, you're not alone

**Comment by Tom5959** [comment]
same here ,hope can update that soon

**Comment by Korozion** [comment]
Solution found on Discord server [https://discord.gg/C8nnf5kvhv] (thank you @Johnthen1 )
In SketchfabRipper folder, open SketchfabRipper.ini with notepad and replace :
`sfConfig= ...`
with
`sfConfig=wgmet7yBv1pVw9b1ZoNns2Yt8G4WNQ6xdsqyWUPbHwLgL0yqbre0lS4EdRb0R4Ud6fHYwHlxM9pEI6z50yUfay66kmvI5E7WvedLi2+wfI59LjyOBye7SGEafPgL6mfy`
and the error disappears. 
This only fixes the model importation, not the textures.

**Comment by yogi0172** [comment]
> 

ya Plz Someone update soon very very important

**Comment by Tom5959** [comment]
> 

it is still can not work,even only the model,show the same problem

**Comment by moumouh206** [comment]
this the new enryption key 👍 
sfConfig=CYlPxwS2rhvwUHAmPjCQqxOnlbeTp6DPO911eSF/opKaTx3I/8YloXNTO8cYpezO9NE4z4CNHkFXEFEyqWG7Jye7s3Ow4LgooUVPCWfiTjcrkbCzgu7FSLgqGUwtjGYr

**Comment by Goldy-the-fox** [comment]
> 

heeeyyy it works! (well atleast as of this comment)

**Comment by Goldy-the-fox** [comment]
broke again

**Comment by moumouh206** [comment]
Open the sketchfab-ripper.ini file in same directory with any text editor and replace the key "sfConfig" with this new encryption key 👍 sfConfig=iqmKWuccsR1DWJLcPTT92OnLU1X94yCrmo5gswmXJ9Y7ADy3XoAfyXab4HFrkUs4yqxZb8ligH6eEMMZvVagj3UPESR0fhbNI4W6r/ezBVQuY86q5hsFUDpTc9UERZGZ

**Comment by JSH32** [comment]
Broke again

**Comment by wrrrrrrffr** [comment]
broke

**Comment by moumouh206** [comment]
> broke

sfConfig=DXb9Nw3w+U9HBtwmZeTs0VU3LJ4qjqx8uB1ALqE2JF+I38b16SGRzC12O0gqzK1LwaL08PTP3qtbnxeUXV4jP/sRO2De4OLpD5uFmjm025Z6bzOM568SJjgaVAx2vRQ2

**Comment by jkattr** [comment]
![image](https://user-images.githubusercontent.com/70831738/170328052-99bc720f-b31b-4274-bf7a-a259335114ee.png)
tried all keys, still get an error message

**Comment by moumouh206** [comment]
sfConfig=tZE+n6VM9ceEz60a4qQ0gkJhHqQnKuvisKJTLXwgFQNARFX2//O/cD7EWPlLoztfbapQjLWDOW9ZzsMC00hJtCZgtFdd9BhB1u+F59EY+/aNU9d2TMP9RdDEZF8uRt6c

**Comment by raghav-kukreti** [comment]
hey @moumouh206 how do you come across these keys, since I am assuming they're used to decrypt *.binz

**Comment by moumouh206** [comment]
> hey @moumouh206 how do you come across these keys, since I am assuming they're used to decrypt *.binz

The key has nothing to do with sketchfab ,the software you're using (SketchfabRipper) is paid one and has some security key to work, and sometimes they change their security key, basically you need to crack it to find the key

**Comment by InnovatorNZ** [comment]
Seems the key is invalid again, still getting the unexpected character error

**Comment by raghav-kukreti** [comment]
@InnovatorNZ yeah, if you try to find the discord of the OG sketchfab-ripper, you'll find a way to extract keys as well, takes sometime to figure out, but is worth it.
